### DISCUSSION TITLE
Best way to use a pinned release of controller

### DISCUSSION THREAD

**Discussion by rahul9170** [actions/actions-runner-controller#1405] [Questions] [answered]
Hello,

I wanted to know the best way to reuse a specific controller release in my own repo. I don't want all the source code to be there as I would just be interested in the helm charts I guess. I would also want to do some customisations to the values.yaml as per my own requirement. May I please know what are the best git practices we follow for this?

Regards,
Rahul Sharma

**Comment by mumoshu** [comment, marked as answer]
@rahul9170 Hey! Well I don't think there's any ARC-specific practices about your requirement, but generally speaking, you could pin controller version via specifying the `image.tag` helm value, in case you use helm for deployment.
It should also be a good idea to pin the chart version if you need, by specifying `--version` flag to your helm-upgrade command.
